server/src/controller/carrito.ts

```diff
@@ -56,15 +57,15 @@
 
       return productos.create(producto)
     } catch (err) {
-      console.log('[falla al guardar]', err)
+      logger.info('[falla al guardar]', err)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/c0dejohn/e-commerce-backend/pullRequest?prid=8292833)
```suggestion
      logger.info("[falla al guardar]", err)
```